src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1259,19 +1259,19 @@ public function renderFieldAction($productId, $step, $fieldName)
     /**
      * @param Request $request
      *
-     * @return Response
+     * @return JsonResponse
      */
-    public function searchProductsAction(Request $request): Response
+    public function searchProductsAction(Request $request): JsonResponse
     {
         try {
-            $searchPhrase = $request->query->get('search_phrase');
+            $searchPhrase = $request->query->get('q');
```

Comment:
Isn't it a BC break ? This is probably used in other places, I would be surprised this code change does not break other BO pages 🤔

Reply:
I doubt it was used, I implemented this action in ProductController with previous pr's of Order page migration steps.

Reply:
It's not a BC to change variable value, It's not related to public API.

Reply:
Thanks for answer, everything's alright !